### PR TITLE
Fixes violations of and reenables the rubocop Style/EmptyLines check.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Fixes generated by "bundle exec rubocop --only Style/EmptyLines --auto-correct".

### CODE REVIEW

**Comment by breville** [comment]
hrm, sometimes i like having two blank lines for bigger breaks :)  but if we've decided that we don't want that, then this change seems to do the job.

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -203,11 +203,6 @@ Style/EmptyElse:
...
   Enabled: false
 
-# Offense count: 142
-# Cop supports --auto-correct.
-Style/EmptyLines:
-  Enabled: false
-
 # Offense count: 20
 # Cop supports --auto-correct.
 Style/EmptyLinesAroundAccessModifier:
```
Comment (breville) [inline .rubocop_todo.yml:208]: is it actually reenabled here?

Comment (ashercodeorg) [inline .rubocop_todo.yml:208]: Yes. The rubocop_todo is essentially a blacklist of lint rules. By removing it from the blacklist, it reverts to the default [whitelist].

**Comment by ashercodeorg** [comment]
In my world view, a double line break probably means a high level comment would be appropriate.

**Comment by ashercodeorg** [comment]
That said, I'm happy to not enable this check if you (others) feel the cost-benefit of the rule is negative.

**Comment by breville** [comment]
I don't have strong feelings in the style discussions, but I think worth floating this with the whole team in either chat room or weekly meeting, just to make sure there's some consensus.

**Comment by ashercodeorg** [comment]
Hearing no disagreement after querying the developers room, I'm taking lack of pushback to mean this should ship.

**Pull request merged**